.github/workflows/verify-push.yaml

```diff
@@ -84,6 +86,16 @@ jobs:
             spring-petclinic-*.jar
           port: 8080
           oci: 'true'
+        env:
+          STORAGE_OPTS: "overlay.mount_program=/usr/bin/fuse-overlayfs"
+
+      # Authenticate to container image registry to push the image
+      - name: Podman Login
```

Comment:
We have to have workflow tests that use username & password too, so we should have two workflows (or jobs), one with login and one with PTR username/password.

Reply:
Yeah, I have left one workflow with username/password approach. This one https://github.com/redhat-actions/push-to-registry/blob/main/.github/workflows/multiple-build.yaml
May be this is hard to figure out for users, shall I add new example?

Reply:
yeah, maybe an `login-example` and just `example`